bar_app/serializers.py

```diff
@@ -0,0 +1,169 @@
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+from django.contrib.auth.models import User
+from .models import *
```

Comment:
I wouldn't recommend to blindly import everything from a module in general. Leave it as is